tests/gnrc_lorawan/Makefile

```diff
@@ -0,0 +1,71 @@
+include ../Makefile.tests_common
+
+USEMODULE += od
+USEMODULE += shell
+USEMODULE += shell_commands
+USEMODULE += ps
+USEMODULE += xtimer
+USEMODULE += random
+USEMODULE += gnrc_netdev_default
+USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_lorawan
+USEMODULE += gnrc_pktbuf_cmd
+USEMODULE += gnrc_netapi_callbacks
+USEMODULE += netdev_layer
+USEMODULE += gnrc_neterr
+USEMODULE += gnrc_netreg
+USEMODULE += hashes
+USEMODULE += crypto
```

Comment:
I guess some of the modules above should be made dependencies of `gnrc_lorawan` and thus are not required here, i.e. `crypto` is not used in the test anywhere

Reply:
fixed